src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/EnvelopeHandler.java

```diff
@@ -35,12 +35,15 @@ public void handleEnvelope(Envelope envelope) {
 
                 if (caseDetailsFound.isPresent()) {
                     CaseDetails existingCase = caseDetailsFound.get();
-                    evidenceAttacher.attach(envelope, existingCase);
-                    paymentsProcessor.createPayments(envelope, existingCase.getId(), false);
+                    boolean docsAttached = evidenceAttacher.attach(envelope, existingCase);
+                    if (docsAttached) {
+                        paymentsProcessor.createPayments(envelope, existingCase.getId(), false);
+                    } else {
+                        createExceptionRecord(envelope);
```

Comment:
imho worth logging here why this happened as this is different from regular path to ER creation